warehouse/query-core/src/main/java/datawave/query/tables/edge/EdgeQueryLogic.java

```diff
@@ -148,6 +162,78 @@ public GenericQueryConfiguration initialize(Connector connection, Query settings
         return cfg;
     }
     
+    public String getJexlQueryString(Query settings) throws datawave.query.language.parser.ParseException {
+        // queryString should be JEXl after all query parsers are applied
+        String queryString;
+        String originalQuery = settings.getQuery();
+        
+        originalQuery = this.expandQueryMacros(originalQuery);
```

Comment:
add this property to edgequeries?

Reply:
fyi i changed the logic in `getJexlQueryString` from shardquerylogic instead of copying it because i think it's more readable this way. One could argue that we should pop the logic out of shardquerylogic and share it between the two. they might be right. but i am not sure.

Reply:
Worth opening some tickets for these.  I'd say they're out of the scope of this ticket.